docs/architecture/adr-027-deterministic-protobuf-serialization.md

```diff
@@ -216,9 +228,20 @@ for all protobuf documents we need in the context of Cosmos SDK signing.
 - When implementing transaction signing, the encoding rules above must be
   understood and implemented.
 - The need for rule number 3. adds some complexity to implementations.
+- Some data structures may require custom code for serialization. Thus
+  the code is not very portable - it will require additional work for each
+  client implementing serialization for not compatible data structures.
 
 ### Neutral
 
+
+### Usage in SDK
+
+For the reasons mentioned above (negative section) we prefer to keep workarounds
+for shared data structure. Example: aforementioned `TxRaw` is using a raw bytes
+as a workaround. This frees them from a risk of wrongly implementing this
+standard on top of their favorite protobuf library.
```

Comment:
I would reword to "This allows them to use any valid Protobuf library." or something, the whole point is to avoid having them implement this standard.

Reply:
thanks

Reply:
updated